### PR TITLE
Add scene support to WMS WebControl pro

### PR DESCRIPTION
**Pull request by mback2k**
## Proposed change

Add scene support to WMS WebControl pro

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/126081
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Added support for "scenes" as a platform alongside "covers."
	- Introduced a new section titled "Scenes" detailing activation capabilities.

- **Documentation**
	- Expanded support information to reference both "covers" and "scenes."
	- Updated formatting of the `ha_release` field for consistency.

### CODE REVIEW

File: source/_integrations/wmspro.markdown [hunk 1]
```diff
@@ -34,3 +35,6 @@ The WMS WebControl pro *may* also be discovered on your local network via DHCP.
 
 - *Patio awnings* can be opened, closed, set to a certain position and their movement stopped.
 - The integration and library *may* already support other types of awnings with a single motor.
+
+## Scenes
```
Comment (c0ffeeca7) [inline source/_integrations/wmspro.markdown:41]:
```suggestion
## Scenes

```

**Review by c0ffeeca7 — DISMISSED** [review]
Thank you, @mback2k :+1:

**Review by joostlek — DISMISSED** [review]
There's a merge conflict

**Comment by mback2k** [comment]
> There's a merge conflict

Rebased and solved.

**Review by c0ffeeca7 — APPROVED** [review]

**Pull request merged**